cli/command/image/load.go

```diff
@@ -35,12 +40,13 @@ func NewLoadCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	flags.StringVarP(&opts.input, "input", "i", "", "Read from tar archive file, instead of STDIN")
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Suppress the load output")
+	flags.StringVarP(&opts.name, "name", "n", "", "Name to use when loading OCI image layout tar archive")
+	flags.StringSliceVar(&opts.refs, "ref", []string{}, "References to use when loading an OCI image layout tar archive")
```

Comment:
@runcom Should we release on rc4? There are few other changes that are in the pipe that are needed to get this through.